src/murmur/Server.cpp

```diff
@@ -568,6 +589,16 @@ void Server::setLiveConf(const QString &key, const QString &value) {
 			mpsc.set_welcome_text(u8(qsWelcomeText));
 			sendAll(mpsc);
 		}
+	} else if (key == "welcometextfile") {
+		QString filename = !v.isNull() ? v : Meta::mp.qsWelcomeTextFile;
+		if (filename != qsWelcomeTextFile) {
+			qsWelcomeTextFile = filename;
+			MumbleProto::ServerConfig mpsc;
+			loadWelcomeTextFromFile();
+			mpsc.set_welcome_text(u8(qsWelcomeText));
+//			mpsc.set_welcome_text_file(u8(qsWelcomeTextFile));
```

Comment:
The client doesn't need to know about the file the welcome message was read from (and probably also shouldn't as information on which files exist on the server might not be something you want to share with all clients). Thus this line can be removed (imo).

Reply:
I see. Wasn't too sure what this code is doing if I'm honest :sweat_smile: